Makefile

```diff
@@ -132,12 +133,21 @@ bin/golangci-lint:
 	mkdir -p bin
 	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | BINARY=golangci-lint bash -s -- v${GOLANGCI_VERSION}
 
-.PHONY: lint lint-fix
+bin/gofumpt:
+	mkdir -p bin
+	test -f bin/gofumpt-$(GOFUMPT_VERSION) || curl -sLo bin/gofumpt-$(GOFUMPT_VERSION) https://github.com/mvdan/gofumpt/releases/download/v$(GOFUMPT_VERSION)/gofumpt_v$(GOFUMPT_VERSION)_$(OS_NAME)_$(PLATFORM_NAME)
+	chmod u+x bin/gofumpt-$(GOFUMPT_VERSION)
+	ln -s gofumpt-$(GOFUMPT_VERSION) bin/gofumpt
+
+.PHONY: lint lint-fix format
 lint: ## Run linter.
-	golangci-lint run
+	bin/golangci-lint run
 
 lint-fix: ## Fix lint violations.
-	golangci-lint run --fix
+	bin/golangci-lint run --fix
+
+format:
```

Comment:
How it works if a dev forgets to call the format command? Will the CI pipeline fail? If we want to use gofumpt as the main formatter, we need to enforce it by providing additional CI checks (if it doesn't already work this way).

Reply:
The action verification is yet to be done